end_to_end_tests/test-ubuntu-16.04.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 # Starts an Ubuntu 16.04 Docker container and runs the Yarn end-to-end test on it
 set -ex
-./data/start-ubuntu.sh ubuntu:16.04
+cd data 
+./run-ubuntu.sh
```

Comment:
Just call `run-ubuntu.sh` directly (in `config.yml`) rather than modifying this script. This script should remain as-is, as it's still useful when running the test suite on your own computer.